<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPlayerDialog</class>
 <widget class="QDialog" name="addPlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Player</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="obligatoryTab">
      <attribute name="title">
       <string>Obligatory</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="dniLabel">
         <property name="text">
          <string>DNI</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dniBox">
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="surnameLabel">
         <property name="text">
          <string>Surname</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="surnameBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="codeLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="codeBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="betsTab">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Bets</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="typeBetLabel">
         <property name="text">
          <string>Bet Type (1-4):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="typeBetBox"/>
       </item>
       <item>
        <widget class="QLabel" name="valueBetLabel">
         <property name="text">
          <string>Bet:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="valueBetBox"/>
       </item>
       <item>
        <widget class="QLabel" name="cuantBetLabel">
         <property name="text">
          <string>Money to bet:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cuantBetBox"/>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="addBet">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Add Bet</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="optionalTab">
      <attribute name="title">
       <string>Optional</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="addressBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cityLabel">
         <property name="text">
          <string>City</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="cityBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="stateLabel">
         <property name="text">
          <string>State</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="stateBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="countrLabel">
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="countryBox"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addPlayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addPlayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
